src/components/tool_tip/tool_tip.tsx

```diff
@@ -286,6 +295,7 @@ export class EuiToolTip extends Component<Props, State> {
       content,
       title,
       delay,
+      display,
```

Comment:
Needs a default so that it shows up in the props table.
```suggestion
      display = 'inlineBlock',
```